frontends/web/src/routes/account/receive/receive.tsx

```diff
@@ -246,13 +250,10 @@ class Receive extends Component<Props, State> {
                 </div>
                 <CopyableInput disabled={!enableCopy} value={address} flexibleHeight />
                 { this.availableScriptTypes.length > 1 && (
-                      this.availableScriptTypes.map(scriptType => (
-                          <p className={style.changeType} onClick={() => this.switchToScriptType(scriptType)}>
-                              {t(`receive.scriptType.${scriptType}`)}
-                              { scriptType === receiveAddresses[addressType].scriptType && ' (active)' }
-                          </p>
-                      ))
-                  )}
+                    <button className={style.changeType} onClick={this.nextAddressType}>
+                        {t(`receive.scriptType.${receiveAddresses[this.getNextAddressTypeIndex(addressType)].scriptType}`)}
```

Comment:
or just `${this.availableScriptTypes[this.getNextAddressTypeIndex(addressType)]}`? less indirection.

Reply:
I added your commit and applied this here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1639